src/components/Paginator/scss/Paginator.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../../scss/index';
+
+/*------------------------------------*\
+    $PAGINATOR
+\*------------------------------------*/
+
+  .paginator {
+    @extend %list--reset;
+    margin: 1rem 0 1rem 0;
+    display: flex;
+  }
+
+    .paginator__item {
+      padding: 0 1rem 0 1rem;
+    }
+
+      .paginator__item__btn {
+        color: black;
+
+        &:active,
+        &:hover {
+          color: red;
+        }
+        &:disabled{
+          color: grey;
+          pointer-events: none;
+        }
```

Comment:
shouldn't these only exist in the storybook's css?

Reply:
not sure I understand your viewpoint

Reply:
I'm suggesting that these are not "functional" styles (colors etc.) and will need to be overridden usage. Just seems strange to include this when (for example) we already have sensible defaults in our projects (which this would override)